stable_baselines3/ppo/ppo.py

```diff
@@ -296,6 +305,38 @@ def train(self, n_epochs: int, batch_size: int = 64) -> None:
         if hasattr(self.policy, 'log_std'):
             logger.logkv("std", th.exp(self.policy.log_std).mean().item())
 
+        # TODO extract learning rate from optimizer?
+        if self.tb_writer is not None:
```

Comment:
my idea was to avoid duplicating the logger and do something similar to https://github.com/hill-a/stable-baselines/blob/master/stable_baselines/logger.py#L218 or https://github.com/rlworkgroup/dowel/blob/master/src/dowel/tensor_board_output.py#L31

Reply:
Agreed. Didn't know about it :)

Reply:
I got it working. So 
```python
logger.logkv("loss/entropy_loss", np.mean(entropy_losses))
```
will log to both stdout and tensorboard. However, I can't figure out how to send logs to only one of the outputs of the `Logger`, e.g. 
```python
logger.logkv("iterations", iteration)
```
Any ideas?

Reply:
maybe additional keyword argument to `logkv` ?

Reply:
Makes sense 👍

Reply:
How does this look?
```python
def logkv(key: Any, val: Any, exclude: Optional[List[str]]) -> None:
    """
    Log a value of some diagnostic
    Call this once for each diagnostic quantity, each iteration
    If called many times, last value will be used.

    :param key: (Any) save to log this key
    :param val: (Any) save to log this value
    :param exclude: (List[str]) list of outputs to be excluded
    """

    Logger.CURRENT.logkv(key, (val, exclude))
```